Added steps to add/remove custom device (#2925)
MicrosoftDocs/edge-developer: microsoft-edge/devtools-guide-chromium/device-mode/index.md

   ![Adding a custom device](./index-images/device-mode-toggle-device-toolbar-settings-emulated-devices-add.png)

> 1. Click **Add** to add the new custom device. The new device should now be available in the **Device** list.

Reviewer: ```suggestion
1. Click **Add** to add the new custom device. The new device is now available in the **Device** list.
```
simplified wording, optimistic (happy path; assume success) wording
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/should-vs-must

Reviewer: applied locally in VS Code

To remove a custom device:

> 1. Under **Emulated Devices**, in the **Devices** list, hover over the custom device name and click the trash icon to delete the custom device.

Reviewer: ```suggestion
1. Under **Emulated Devices**, in the **Devices** list, hover over the custom device name, and then click the trash icon.
```
neither term is in style guide
https://learn.microsoft.com/en-us/search/?scope=Style%20guide&terms=trash
https://learn.microsoft.com/en-us/search/?scope=Style%20guide&terms=wastebasket

https://ahdictionary.com/word/search.html?q=trashcan
